### ISSUE TITLE
Quiz questions: "Check" and "Show answer" should be <button> not <div> (keyboard interactivity)

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/ocw-hugo-themes#1252]
## Expected Behavior
The "Check" and "Show answer" buttons should
- be focusable
- fire click event handlers when pressing spacebar/Enter

I.e., the should behave like normal browser buttons.

## Current Behavior
The "Check" and "Show/Hide answer" buttons on quiz questions are normal divs; they cannot be focused by keyboard users or interacted with.

## Steps to Reproduce
See https://ocw.mit.edu/courses/15-071-the-analytics-edge-spring-2017/pages/logistic-regression/the-framingham-heart-study-evaluating-risk-factors-to-save-lives/quick-question-225/

## Possible Solution
Use normal `<button>` elements.
